components/src/web-components/visualization/gs-prevalence-over-time.tsx

```diff
@@ -143,6 +144,26 @@ export class PrevalenceOverTimeComponent extends PreactLitAdapterWithGridJsStyle
     @property({ type: Object })
     pageSize: boolean | number = false;
 
+    /**
+     * The maximum value for the y-axis on all graphs in linear view.
+     * If set to a number, the maximum value is set to this number.
+     * If set to `maxInData`, the maximum value is set to the maximum value in the data.
+     * If set to `limitTo1`, the maximum value is set to the lower value of 1 or the maximum value in the data.
```

Comment:
For me, this would be easier to understand:
```suggestion
     * If set to `limitTo1`, the maximum value is set to `min(1, the maximum value in the data)`.
```